docs/_blog/2020-04-16-working-with-private-registries.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: "Working with Private Registries"
+categories: blog
+author_picture: images/blog/author_icon_andrew_mak.jpeg
+author_url: 
+author: Andrew Mak
+blog_description: "Need a specific capability in your application? Chances are there's a code library or package out there that already does the job."
+permalink: working-with-private-registries
+duration: 10 minutes
+tags: [private, registry, registries, repository, repositories]
+---
+
+Need a specific capability in your application? Chances are there's a code library or package out there that already does the job. Such is the norm for modern-day application development, which is part writing your own code and part reusing code pulled from a registry or repository somewhere.
+
+In most cases, these 3rd-party bits of code are hosted on well-known public registries and repositories, and build and package tools such as `npm` and `mvn` (Maven) helps you manage these dependencies and makes pulling the code down for use simple. However, if you work in a medium or large-sized organization, you may be required to pull 3rd-party code from private servers within the organization--it's also simple to do, as long as you have proper configuration in place.
```

Comment:
- Should we say, "`npm` (Node Package Manager)"?
- For correct subject/verb agreement, say, "help" and "make". Tools...help and make.
- "might" instead of "may" to indicate possibility.

Reply:
Changed